perma_web/api/tests/test_current_user_authorization.py

```diff
@@ -89,7 +89,7 @@ def test_should_provide_different_capture_jobs_data_relative_to_user(self):
         vm_data = self.successful_get(self.detail_url + 'capture_jobs/', user=self.org_user)
         reg_data = self.successful_get(self.detail_url + 'capture_jobs/', user=self.regular_user)
 
-        self.assertEqual(vm_data.keys(), reg_data.keys())
+        self.assertEqual(list(vm_data.keys()), list(reg_data.keys()))
```

Comment:
I don't see why this works in 3.5, when dictionaries aren't ordered by default, but.......